packages/zowe-explorer/src/utils/SettingsConfig.ts

```diff
@@ -176,7 +176,40 @@ export class SettingsConfig {
         }
 
         if (workspaceIsMigrated) {
-            await SettingsConfig.setDirectValue(globals.SETTINGS_VERSION, SettingsConfig.currentVersionNumber, vscode.ConfigurationTarget.Workspace);
+            await SettingsConfig.setDirectValue(globals.SETTINGS_OLD_SETTINGS_MIGRATED, true, vscode.ConfigurationTarget.Workspace);
```

Comment:
Does this need to be `ZoweLocalStorage.setValue` instead?

Reply:
For this case no, since local storage works at a global level and this setting is at a workspace level. We could perhaps in the future consider adding a workspace storage option to the ZoweLocalStorage class :)